src/main/scala/org/zalando/spearheads/innkeeper/services/RoutesService.scala

```diff
@@ -33,47 +39,45 @@ class RoutesService @Inject() (implicit val executionContext: ExecutionContext,
     }
   }
 
-  def findModifiedSince(localDateTime: LocalDateTime): Source[Route, Unit] = {
+  def findModifiedSince(localDateTime: LocalDateTime): Source[RouteOut, Unit] = {
     Source(
-      routesRepo.selectModifiedSince(localDateTime).mapResult { row =>
-        row.id.map { id =>
-          Route(
-            id = id,
-            route = row.routeJson.parseJson.convertTo[NewRoute],
-            row.createdAt,
-            row.deletedAt
-          )
+      routesRepo.selectModifiedSince(localDateTime).mapResult { routeRow =>
+        routeRow.id.map { id =>
+          routeRowToRoute(id, routeRow)
         }
       }
     ).mapConcat(_.toList)
   }
 
-  def allRoutes: Source[Route, Unit] = {
-    Source(routesRepo.selectAll.mapResult { row =>
-      row.id.map { id =>
-        Route(
-          id,
-          row.routeJson.parseJson.convertTo[NewRoute],
-          row.createdAt,
-          row.deletedAt
-        )
+  def allRoutes: Source[RouteOut, Unit] = {
+    Source(routesRepo.selectAll.mapResult { routeRow =>
+      routeRow.id.map { id =>
+        routeRowToRoute(id, routeRow)
       }
     }).mapConcat(_.toList)
   }
 
-  def findRouteById(id: Long): Future[Option[Route]] = {
+  def findRouteById(id: Long): Future[Option[RouteOut]] = {
     routesRepo.selectById(id).flatMap {
       case Some(routeRow) => rowToEventualMaybeRoute(routeRow)
       case None           => Future(None)
     }
   }
 
-  private def rowToEventualMaybeRoute(routeRow: RouteRow): Future[Option[Route]] = routeRow.id match {
-    case Some(id) => Future(Some(Route(id,
-      routeRow.routeJson.parseJson.convertTo[NewRoute],
-      routeRow.createdAt,
-      routeRow.deletedAt)))
-    case None => Future(None)
+  private def rowToEventualMaybeRoute(routeRow: RouteRow): Future[Option[RouteOut]] = routeRow.id match {
+    case Some(id) => Future(Some(routeRowToRoute(id, routeRow)))
+    case None     => Future(None)
+  }
+
+  private def routeRowToRoute(id: Long, routeRow: RouteRow): RouteOut = {
```

Comment:
Why is this method not returning in `Option[RouteOut]`? If it fails, it could just return a None.